### DISCUSSION TITLE
how to create / get a standard.mm

### DISCUSSION THREAD

**Discussion by EdoFro** [freeplane/freeplane#277] [Help] [answered]
originally posted in [SF Forum](https://sourceforge.net/p/freeplane/discussion/758437/thread/a39878155a/):

> I don't have a standard.mm in my installation, Where can I get one or how to create one? But still, when creating a new mindmap it asks which template I want to use. Where are these stored?
Thanks for any help
Martin

**Comment by EdoFro** [comment, marked as answer]
you can find where the templates are by clicking `File->New map from template…`

it opens a dialog. If you select **Standard templates** it will go to the default folder and show the standard templates

![image](https://user-images.githubusercontent.com/35700575/160222266-e09be7f0-eae4-4b78-8a8d-3066a28e9611.png)

------

Also, you can change  your default template in preferences:

![image](https://user-images.githubusercontent.com/35700575/160222327-024f28a3-c2d5-4a44-8a81-3f8697b6e7d9.png)

bye,

edo

**Reply by MrMfromK** [reply]
Hi Edo,
thanks.
it looks a little different under windows, the selection for standard templates is at the top of the window. But I finally found it.
